content/docs/manage/organizations.md

```diff
@@ -165,13 +165,262 @@ As the Admin for the organization account:
 
 For detailed information on pricing and plans, refer to [Neon plans](/docs/introduction/plans).
 
+## Managing projects using the Neon API
+
+In the Neon API, the `org_id` represents the unique identifier for your organization. Use this ID to manage and interact with your organization's projects, making sure that your API requests are scoped to the right organization.
+
+When creating a new project, include `org_id` in your `POST` request to ensure the project gets associated with your organization. Otherwise, the project will be created under your personal account.
+
+You can also use the `org_id` parameter in `GET` requests to:
+
+- List all projects that belong to your organization
+- Get consumption metrics for your organization
+- List of all organizations that you belong to (via your personal account)
+
+### Finding your org_id
+
+While you can't get your `org_id` directly in the Neon Console (it's not yet available during private preview), if you navigate to your organization, you can find the ID in the console URL:
```

Comment:
maybe worth mentioning org_id always starts with "org-". Might be easier for customers to understand what it is

Reply:
Good point. Added.